src/guide-start.md

```diff
@@ -63,7 +63,29 @@ All generators implement the [`Rng`] trait, which provides the [`gen`],
 Rand provides functionality on iterators and slices via two more traits,
 [`IteratorRandom`] and [`SliceRandom`].
 
+## Fixed seed RNGs
 
+You may have noticed the use of `thread_rng()` above and wondered how to
+specify a fixed seed. To do so, you need to specify an RNG then use a method
+like [`seed_from_u64`] or [`from_seed`].
+
+We use `ChaCha8Rng` below because it is fast and portable with good quality.
+See the [RNGs] section for more RNGs, but avoid `SmallRng` and `StdRng` if you
+care about reproducible results.
+
+```rust,editable
+extern crate rand;
+extern crate rand_chacha;
```

Comment:
Aren't we using Rust 2018?

Reply:
Yes, and? Unlike with a real crate we have no `Cargo.toml` here to describe dependencies.

Reply:
Fair enough.